src/components/compound/Styles.module.css

```diff
@@ -1,5 +1,5 @@
-.pageLayout {
+/* .pageLayout {
```

Comment:
Remove commented code. If this is deprecated code. Uncommented but don't call it anywhere

Reply:
Pushed a couple of commits to fix this and another issue presented while doing the PR.